runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/runtime/Platform.java

```diff
@@ -409,6 +409,15 @@ public static boolean isMac() {
 	 */
 	public static final String ARCH_AARCH64 = "aarch64";//$NON-NLS-1$
 
+	/**
+	 * Constant string (value {@code aarch64} indicating the platform is running on
```

Comment:
The value of course should have been `riscv64`. This is fixed in
- https://github.com/eclipse-platform/eclipse.platform/pull/1578